lib/analytics/course_articles_csv_builder.rb

```diff
@@ -31,6 +31,7 @@ def set_articles_edited
     @course.all_revisions.includes(article: :wiki).map do |edit|
       article_edits = @articles_edited[edit.article_id] || new_article_entry(edit)
       article_edits[:characters][edit.mw_rev_id] = edit.characters
+      article_edits[:references] = edit.references_added
```

Comment:
Whoops... I merged this a little too quickly. This will overwrite the previous value, right? Instead of providing a total across all edits?

Reply:
I'll poke at it today and do a fix if necessary.